connectors/httpclient/src/main/java/org/openstack4j/connectors/httpclient/HttpResponseImpl.java

```diff
@@ -123,8 +125,13 @@ public String header(String name) {
     @Override
     public <T> T readEntity(Class<T> typeToReadAs) {
         HttpEntity entity = response.getEntity();
+        if (entity == null) {
```

Comment:
Why not use checkNotNull(entity) here, this will make it more compact and readable, what do you think?